sys/riotboot/flashwrite.c

```diff
@@ -162,6 +191,7 @@ int riotboot_flashwrite_finish_raw(riotboot_flashwrite_t *state,
 #if CONFIG_RIOTBOOT_FLASHWRITE_RAW
     memcpy(state->firstblock_buf, bytes, len);
     flashpage_write(slot_start, state->firstblock_buf, RIOTBOOT_FLASHPAGE_BUFFER_SIZE);
+    res = 0;
```

Comment:
This one should not be needed, I think it is already fixed upstream, should go away with a rebase.